### ISSUE TITLE
[Feature Request] Multi Subtitles Support

### ISSUE THREAD

**Issue by creeper451** [hardcore-sushi/DroidFS#125]
When I try to open a mkv file with multiple subtitles i can't change the default one nor can i open a custom srt/ass subtitle file
![Screenshot_20220429-102435_1](https://user-images.githubusercontent.com/92668459/165901871-355d8d99-2146-49c6-890a-67231977ea19.png)

**Comment by hardcore-sushi** [comment]
Isn't that one of the features requested in #88?

**Comment by creeper451** [comment]
yes but they didn't say anything about mkv files mkv files can conatain multiple subtitles if i wanna switch between english and spanish for example
